region/norfair/east/Double Chamber.json

```
@@ -1527,6 +1527,55 @@
         "IBJ past the Ripper or kill it. The Kago bugs will not spawn until Samus exits G-mode, so it is fine to kill it or IBJ from on top of it."
       ]
     },
+    {
+      "link": [2, 3],
+      "name": "R-Mode Standup Clip",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Varia",
+        "Morph",
+        "canRModeStandupClip",
+        {"disableEquipment": "ETank"},
+        {"resourceAvailable": [{"type": "Energy", "count": 50}]},
+        {"partialRefill": {"type": "ReserveEnergy"}},
```

```suggestion
        {"partialRefill": {"type": "ReserveEnergy", "limit": 100}},
```
maybe something like this and then ~200 heatFrames at the end of the strat?